circle.yml

```diff
@@ -1,3 +1,7 @@
+machine:
+  node:
+    version: 6.11.5
```

Comment:
shouldn't this go into a separate commit? ;)

Reply:
And that is my PR https://github.com/coala/coala-atom/pull/83
He didn't rebase properly